### DISCUSSION TITLE
Using `eslint-env` comments with the flat config

### DISCUSSION THREAD

**Discussion by fasttime** [eslint/eslint#17149] [Show and tell]
https://www.npmjs.com/package/eslint-plugin-eslint-env

A processor that translates `eslint-env` config comments to `global` comments under the hood, so they can be used with the flat config.

I wrote this mostly to test the new config system on my existing projects without having to edit the source files (yet).

The same effect could be achieved more efficiently with a custom parser, but that would likely stop working once `eslint-env` comments start to throw errors, as planned for the last phase of the [implementation](https://github.com/eslint/eslint/issues/13481). This is why I decided to write a processor.

**Comment by nzakas** [comment]
Interesting!
